src/chart.zoom.js

```diff
@@ -317,16 +317,14 @@ var zoomPlugin = {
 				var tickOptions = scale.options.ticks;
 
 				if (timeOptions) {
-					delete timeOptions.min;
-					delete timeOptions.max;
+					timeOptions.min = scale.originalOptions.time.min;
+					timeOptions.max = scale.originalOptions.time.max;
 				}
 
 				if (tickOptions) {
-					delete tickOptions.min;
-					delete tickOptions.max;
+					tickOptions.min = scale.originalOptions.ticks.min;
+					tickOptions.max = scale.originalOptions.ticks.max;
```

Comment:
This makes only sense if the min/max values are initially set and they should always be in that range.
If you change the range dynamically because some values are out of that min/max-range you always get the initially one after a reset. Is that how it's suppost to be? 
As I understand it, it should reset the zoom to the max and min values of the chart/scale and not some initially wanted one. Maybe if you want to have that, there should be a flag which indicates if the boundaries should be deleted or the initial values are set.
Also a bug is when you change the scale type dynamically, the originalOptions for the scale are vanished and this produces an error.
I think the delete version is the better one.